pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -127,3 +129,88 @@ def close(self) -> None:
         if not self.skip_check_coredump:
             self._check_coredump()
         super().close()
+
+    Case = namedtuple('Case', 'index name type keyword group attribute subcase')
```

Comment:
a list should be named with plurals.

```suggestion
    Case = namedtuple('Case', 'index name type keywords groups attributes subcases')
```

Reply:
or could use https://docs.python.org/3/library/dataclasses.html#dataclasses.dataclass to make your editor/IDE lint with data types.